modules/nlu/src/views/intents/index.jsx

```diff
@@ -258,11 +236,11 @@ export default class IntentsEditor extends React.Component {
         <SplitterLayout secondaryInitialSize={350} secondaryMinSize={200}>
           {this.renderEditor()}
           <div className={style.entitiesPanel}>
-            <EntitiesEditor
+            <Slots
+              ref={el => (this.slotsEditor = el)}
               axios={this.props.axios}
-              ref={el => (this.entitiesEditor = el)}
-              entities={this.state.entities}
-              onEntitiesChanged={this.onEntitiesChanged}
+              slots={this.state.slots}
+              onSlotsChanged={this.onSlotsChanged}
```

Comment:
usually the method doing an operation should be called "handleSlotsChanged", so it's easier to read, like onSlotsChanged={this.handleSlotsChanged}